app/controllers/merchant/items_controller.rb

```diff
@@ -1,17 +1,38 @@
 class Merchant::ItemsController < Merchant::BaseController
 
   def index
-    @merchant = Merchant.first
```

Comment:
thank you sir. meant to ask about this earlier...got quite a few interesting error messages